src/popper/runner_host.py

```diff
@@ -275,10 +275,10 @@ def run(self, step):
         self._spawned_containers.add(container)
 
         cmd = ["podman", "start", "-a", container]
-        _, e, output = HostRunner._exec_cmd(cmd, logging=False)
+        _, e, _ = HostRunner._exec_cmd(cmd)
 
         if e != 0:
-            log.fail(f"{output}")
+            log.fail(f"{e}")
```

Comment:
So there's no need to fail here, this is already handled by the StepRunner ([this function](https://github.com/getpopper/popper/blob/master/src/popper/runner.py#L139-L164)). Instead, this can return `e` directly without checking its value, as there is nothing especial that needs to be done when a failure is encountered

Reply:
Oh oh true! Thanks!!